daemon/network/filter_test.go

```diff
@@ -131,6 +137,16 @@ func TestFilterNetworks(t *testing.T) {
 			err:         "",
 			name:        "global scope filters",
 		},
+		{
+			filter:      trueDanglingFilters,
```

Comment:
Sorry to do this, but do you think we can alter the test to actually test for which networks should be returned? Can you also add a network that is not a built-in but is not dangling?

Reply:
@cpuguy83 Haha, I'll get to it soon :+1:

Reply:
@cpuguy83 Done! :+1: